### PR TITLE
Docs: DBComposite reword to make a little more sense.

### PR DESCRIPTION
**Pull request by LiamKearn**
This makes a little more sense to me since DBComposite isn't actually an `interface`.

I haven't read the code or used this before so could be wrong just offering a quick fix.

### CODE REVIEW

File: src/ORM/FieldType/DBComposite.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 use SilverStripe\ORM\Queries\SQLSelect;
 
 /**
- * Apply this interface to any {@link DBField} that doesn't have a 1-1 mapping with a database field.
+ * Use this template when subclassing any {@link DBField} that doesn't have a 1-1 mapping with a database field.
```
Comment (emteknetnz) [inline src/ORM/FieldType/DBComposite.php:11]:
```suggestion
 * Extend this class when designing a {@link DBField} that doesn't have a 1-1 mapping with a database field.
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]
How about this?

File: src/ORM/FieldType/DBComposite.php (hunk 1 above)
Comment (LiamKearn) [reply src/ORM/FieldType/DBComposite.php]: Makes sense to me interface just threw me off as a newer developer. I'd imagine etiquette is to squash/fixup (if I got those terms right) these two commits but this is as far as time will let me take this PR. Sorry!

**Review by emteknetnz — APPROVED** [review]

**Comment by emteknetnz** [comment]
Thanks!

**Pull request merged**